src/client/app/components/maps/MapViewComponent.tsx

```diff
@@ -2,436 +2,91 @@
-				<FormattedMessage id={buttonMessageId} />
-			</Button>;
-		} else {
-			toggleButton = <div />;
-		}
-
-		if (hasToken()) {
-			return (
-				<div>
-					{formattedCircleSize}
-					{toggleButton}
-				</div>
-			);
-		} else {
-			return (
-				<div>
-					{this.props.map.circleSize}
-					{toggleButton}
-				</div>
-			);
-		}
-	}
-
-	private formatStatus(): string {
-		if (this.props.isSubmitting) {
-			return '(' + this.props.intl.formatMessage({ id: 'submitting' }) + ')';
-		}
-		if (this.props.isEdited) {
-			return this.props.intl.formatMessage({ id: 'edited' });
-		}
-		return '';
-	}
-
-	private toggleDelete() {
-		const consent = window.confirm(`${this.props.intl.formatMessage({ id: 'map.confirm.remove' })} "${this.props.map.name}"?`);
-		if (consent) { this.props.removeMap(this.props.id); }
-	}
-
-	private formatDeleteButton() {
-		const editButtonStyle: React.CSSProperties = {
-			display: 'inline', // or 'none'
-			paddingLeft: '5px'
-		};
-		return <Button style={editButtonStyle} color='primary' onClick={this.toggleDelete}>
-			<FormattedMessage id={'delete.map'} />
-		</Button>;
-	}
-
-	private styleEnabled(): React.CSSProperties {
-		return { color: 'green' };
-	}
-
-	private styleDisabled(): React.CSSProperties {
-		return { color: 'red' };
-	}
-
-	private styleToggleBtn(): React.CSSProperties {
-		return { float: 'right' };
-	}
-
-	private toggleMapDisplayable() {
-		const editedMap = {
-			...this.props.map,
-			displayable: !this.props.map.displayable
-		};
-		this.props.editMapDetails(editedMap);
-	}
-
-	private formatDisplayable() {
-		let styleFn;
-		let messageId;
-		let buttonMessageId;
-
-		if (this.props.map.displayable) {
-			styleFn = this.styleEnabled;
-			messageId = 'map.is.displayable';
-			buttonMessageId = 'hide';
-		} else {
-			styleFn = this.styleDisabled;
-			messageId = 'map.is.not.displayable';
-			buttonMessageId = 'show';
-		}
-
-		let toggleButton;
-		if (hasToken()) {
-			// throw out alert if the admin wants to display uncalibrated map
-			if (!(this.props.map.origin && this.props.map.opposite)) {
-				toggleButton = <Button style={this.styleToggleBtn()} color='primary' onClick={this.notifyCalibrationNeeded}>
-					<FormattedMessage id={buttonMessageId} />
-				</Button>;
-			}
-			// if map is already calibrated, the button will allow it to be displayed
-			else {
-				toggleButton = <Button style={this.styleToggleBtn()} color='primary' onClick={this.toggleMapDisplayable}>
-					<FormattedMessage id={buttonMessageId} />
-				</Button>;
-			}
-		} else {
-			toggleButton = <div />;
-		}
-
-		return (
-			<span>
-				<span style={styleFn()}>
-					<FormattedMessage id={messageId} />
+/**
+ * Defines the map info card
+ * @param props variables passed in to define
+ * @returns Map info card element
+ */
+function MapViewComponent(props: MapViewProps) {
+	const [showEditModal, setShowEditModal] = useState(false);
+
+	useEffect(() => {
+		if (props.isEdited) {
+			//updateUnsavedChanges();
```

Comment:
I think this is no used by OED at this time and probably should go from the page as it seems to be on other pages.